models/event.py

```diff
@@ -214,6 +215,23 @@ def rankings(self):
                 self._rankings = None
         return self._rankings
 
+    @property 
+    def rankings_enhanced(self):
+        if self._rankings is not None and self.year == 2016:
+            self._rankings_enhanced = {}
+            team_number_index = self._rankings[0].index("Team")
```

Comment:
Functionality implemented, screenshot attached. I see two problems with the current implementation:
1) Doesn't look great
2) Not possible to sort by average RS, which seems to be a big part of why users want this

I missed Phil's comment there. I will include that correction in the next commit.

![image](https://cloud.githubusercontent.com/assets/16950452/17161953/886dc250-537f-11e6-8770-19720a33886c.png)